CONTRIBUTING.md

```diff
@@ -7,9 +7,9 @@
 
 ## Running the tests
 
-`./script/test.sh`
+`cargo test`
 
-in particular `./script/tests/test_fixtures.sh` is what I use for ongoing
+in particular `cargo test --test fixtures_test` is what I use for ongoing
```

Comment:
(This whole file probably needs some brushing up at some point, but that's for another day)